### PR TITLE
Remove dependency on variable to pass project id

### PR DESCRIPTION
**Pull request by keshavsoni251199**

### CODE REVIEW

File: UI/K8s/K8s_ui/microblog2/app/cluster/routes.py [hunk 1]
```diff
@@ -45,6 +35,9 @@ def cluster():
                           status='Pending',
                           node_count=form.node_count.data,
                           user_id=userID_exist)
+        session['vm_nodecount'] = cluster.node_count
+        vm_nodecount = session['vm_nodecount']
+        print(vm_nodecount)
```
Comment (pooja1pathak) [inline UI/K8s/K8s_ui/microblog2/app/cluster/routes.py]: remove print statement.

File: UI/K8s/K8s_ui/microblog2/app/cluster/routes.py [hunk 2]
```diff
@@ -56,6 +49,18 @@ def cluster():
 @bp.route('/clusters', methods=['GET'])
 #@login_required
 def clusters():
+    project_id=session['projectid']
+
```
Comment (pooja1pathak) [inline UI/K8s/K8s_ui/microblog2/app/cluster/routes.py]: Remove one blank line

File: UI/K8s/K8s_ui/microblog2/app/vm_node/forms.py [hunk 3]
```diff
@@ -1,56 +0,0 @@
-from flask_wtf import Form
```
Comment (pooja1pathak) [inline UI/K8s/K8s_ui/microblog2/app/vm_node/forms.py:1]: Is this file not required anymore?

File: UI/K8s/K8s_ui/microblog2/app/vm_node/routes.py [hunk 4]
```diff
@@ -13,12 +17,58 @@
...
+        session['Pip'] = Pip1
+        Pip = session['Pip']
     cluster_id = request.args.get('id')
+    class VmNodeCreationForm(Form):
+        vm_name_prefix = StringField('Name', [
+                           Length(max=30, message='max lenth 50 allowed'),
+                           DataRequired(),
+                           #Regexp('^\w+$', message="vm_name_prefix must contain only letters numbers or underscore")
+        ])
+    #vm_ip = TextAreaField(u'vm_ips', [validators.optional(), validators.length(max=300)])
```
Comment (pooja1pathak) [inline UI/K8s/K8s_ui/microblog2/app/vm_node/routes.py]: remove commented code

**Review by pooja1pathak — CHANGES_REQUESTED** [review]

File: UI/K8s/K8s_ui/microblog2/app/cluster/routes.py (hunk 1 above)
Comment (keshavsoni251199) [reply UI/K8s/K8s_ui/microblog2/app/cluster/routes.py]: Changes done

File: UI/K8s/K8s_ui/microblog2/app/cluster/routes.py (hunk 2 above)
Comment (keshavsoni251199) [reply UI/K8s/K8s_ui/microblog2/app/cluster/routes.py]: changes done

File: UI/K8s/K8s_ui/microblog2/app/vm_node/forms.py (hunk 3 above)
Comment (keshavsoni251199) [reply UI/K8s/K8s_ui/microblog2/app/vm_node/forms.py:1]: The file is merged in Vm node/routes

File: UI/K8s/K8s_ui/microblog2/app/vm_node/routes.py (hunk 4 above)
Comment (keshavsoni251199) [reply UI/K8s/K8s_ui/microblog2/app/vm_node/routes.py]: Changes done

**Review by pooja1pathak — APPROVED** [review]

**Review by pooja1pathak — APPROVED** [review]

**Pull request merged**